### DISCUSSION TITLE
popup modal keeps loading without showing login methods

### DISCUSSION THREAD

**Discussion by Felix-Asante** [Web3Auth/Web3Auth#701] [Help]
I don't really know why but for some reasons whenever i call web3auth.logout() and try to trigger the popup modal again, it just keep loading without showing the login methods. but when i use web3auth.logout() again , it sometimes shows the login methods.

can anyone please help me with it. 
![Screen Shot 2022-10-07 at 7 13 59 PM](https://user-images.githubusercontent.com/62264989/194623369-dfe1ee50-3c41-4ff7-a369-8793094c02c7.png)

![Screen Shot 2022-10-07 at 7 14 44 PM](https://user-images.githubusercontent.com/62264989/194623410-4435e2f8-38e9-4ce3-bd4e-ceccfea45252.png)

**Comment by shahbaz17** [comment]
Hey @Felix-Asante 

Which version of `@web3auth/web3auth` are you using? If not the latest version, please try updating it to the latest version, i.e., `2.1.2`, and see if the problem persists.

**Reply by Felix-Asante** [reply]
I am actually using the latest version
"@web3auth/base": "^2.1.3",
"@web3auth/core": "^2.1.3",
"@web3auth/openlogin-adapter": "^2.1.3",
"@web3auth/torus-wallet-connector-plugin": "^2.1.3",
"@web3auth/web3auth": "^2.1.3",
